### PR TITLE
feat: apply stopwords by language

### PR DESCRIPTION
**Pull request by bietkul**
#### What does this do / why do we need it?
https://www.notion.so/appbase/Arc-Suggestions-API-enhancements-2a4c26df77d44a77b751d9e13cf4cb0f

### CODE REVIEW

File: plugins/querytranslate/languages.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+var StemLanguages = []string{
+	"english",
+	"spanish",
+	"french",
+	"russian",
+	"swedish",
+	"norwegian",
+}
+
+// languages supported by stopword package
```
Comment (siddharthlatest) [inline plugins/querytranslate/languages.go]:
```suggestion
// languages supported by stopwords package
```

File: plugins/querytranslate/suggestions.go [hunk 2]
```diff
@@ -468,8 +479,15 @@ func getPredictiveSuggestions(config SuggestionsConfig, suggestions *[]Suggestio
...
 									firstWord := strings.Trim(prefixWords[i:][0], " ")
-									if stopwordsToApply[firstWord] {
-										continue
+									if len(customStopwords) > 0 {
+										if customStopwords[firstWord] {
+											continue
+										}
+									} else {
+										cleanContent := removeStopwords(firstWord, config.Language)
+										if len(strings.Trim(cleanContent, " ")) == 0 {
```
Comment (siddharthlatest) [inline plugins/querytranslate/suggestions.go]:
```suggestion
										if len(strings.TrimSpace(cleanContent)) == 0 {
```

Comment (siddharthlatest) [reply plugins/querytranslate/suggestions.go]: TrimSpace removes all whitespaces instead of explicitly removing only space.

**Review by siddharthlatest — CHANGES_REQUESTED** [review]
@bietkul 

- [ ] Add the documentation reference for the `searchLanguage` property and valid values.
- [ ] It seems that this only applies for suggestions API, it should apply for search API as well.

**Also a question:**
- If searchLanguage isn't explicitly passed in RS API, will this take the search relevance's language for stopwords? It should be this way.

**Comment by bietkul** [comment]
@siddharthlatest Docs PR https://github.com/appbaseio/Docs/pull/246

> If searchLanguage isn't explicitly passed in RS API, will this take the search relevance's language for stopwords? It should be this way.

Yes, review this PR https://github.com/appbaseio-confidential/arc-noss/pull/230

**Review by siddharthlatest — APPROVED** [review]
lgtm (y)

**Pull request merged**